crates/turbopack-ecmascript/tests/tree-shaker/analyzer/multi-export/output.md

```diff
@@ -0,0 +1,230 @@
+# Items
+
+Count: 6
+
+## Item 1: Stmt 0, `VarDeclarator(0)`
+
+```js
+const dog = "dog";
+
+```
+
+- Declares: `dog`
+- Write: `dog`
+
+## Item 2: Stmt 1, `VarDeclarator(0)`
+
+```js
+const cat = "cat";
+
+```
+
+- Declares: `cat`
+- Write: `cat`
+
+## Item 3: Stmt 2, `Normal`
+
+```js
+export { dog as DOG, cat };
+
+```
+
+- Side effects
+- Reads: `dog`, `cat`
```

Comment:
@kdy1 The whole statement is pretty wrong.

It should not have side effects. It should not read vars. It shouldn't exist at all.

Reply:
But if it does not exist, dependency graph will be wrong I think